dev-requirements.txt

```diff
@@ -1,4 +1,4 @@
 cookiecutter>=2.1.0
 pytest
 pytest-black
-mlflow[pipelines]>=1.29.0
+git+https://github.com/mlflow/mlflow.git@4b259e
```

Comment:
Let's change this to `mlflow>=2.0.0`. The pre-release version is already published: https://pypi.org/project/mlflow/2.0.0rc0/